### ISSUE TITLE
Microsoft Teams Rooms for Android Security

### ISSUE THREAD

**Issue by SOM-Scott** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#8318]
Please create another article like this specifically for Android. Given that devices run Android 9.0 out of the box, there are a lot of concerns around security and the devices.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: aa81c0c7-1f2d-09ed-da2d-298aba689e56
* Version Independent ID: 73fd96c0-2b94-d7ea-f900-e88b84344bdd
* Content: [Microsoft Teams Rooms Security - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/rooms/security)
* Content Source: [Teams/rooms/security.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/rooms/security.md)
* Service: **msteams**
* GitHub Login: @dstrome
* Microsoft Alias: **dstrome**

**Comment by dariomws** [comment]
Hi @SOM-Scott, thank you for your feedback.

@dstrome, could you kindly help here? Thanks!

**Comment by dstrome** [comment]
@SOM-Scott can you give examples of the information you'd like to see in an android-specific topic?

**Comment by SOM-Scott** [comment]
We ran into a few issues when deploying the device, and there is no information related to Teams Rooms for Android. Basically, as noted at the top, this page is for Windows, a similar one that touches all the points for Android would be great for security vetting.

Of the most paramount concern that we ran into was that the devices run Android 9.0 (past EOL). There is also no information about how locked down the Kiosk mode is (could very by vendor) or that the camera and touch screen need to talk to each other.
